sklearn/datasets/_base.py

```diff
@@ -292,6 +299,12 @@ def load_wine(return_X_y=False):
 
     (data, target) : tuple if ``return_X_y`` is True
 
+    frame : pandas DataFrame
```

Comment:
I think that we should split `data` and `target` above to state the possible `dtype`:

```
    data : {ndarray, dataframe} of shape (178, 13)
        The data matrix. If `as_frame=True`, `data` will be a pandas DataFrame.

    target : {ndarray, series} of shape (178,)
        The classification labels. If `as_frame=True`, `target` will be a pandas Series.

    as_frame : dataframe of shape (178, 13)
        ...
```

Reply:
This more closely resembles the [downloadable datasets documentation](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html) - it may increase consistency by following the convention of describing bunch attributes explicitly.

Reply:
> describing bunch attributes explicitly.

It is what I meant but I only gave the description for data, target, and frame. Basically I want something similar to `fetch_openml`: https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_openml.html